docs/management/alerting/rule-management.asciidoc

```diff
@@ -0,0 +1,58 @@
+[role="xpack"]
+[[alert-management]]
+=== Managing Rules
```

Comment:
Should we be referring to these as "Alerting Rules" here? 🤔 

I'm not 100% sure when we should use the whole term rather than just "Rules"...

Reply:
Since this is under the `Rules and Connectors` page and there's a sibling page called `Managing Connectors`, I think `Managing Rules` is ok here? I'll let @gchaps be the final judge :)